app/views/layouts/application.html.erb

```diff
@@ -5,7 +5,7 @@
     <!-- As a heading -->
     <nav class="navbar navbar-light bg-light">
       <span class="navbar-brand mb-0 h1">Navbar</span>
-      <a href="https://github.com/arkency/find-open-source-mentor" class="navbar-left"><img src="assets/github.jpg" style="max-width:100px; margin-top: -7px;"></a>
+      <a href="https://github.com/arkency/find-open-source-mentor" class="navbar-left"><img src="assets/images/github.jpg" style="max-width:100px; margin-top: -7px;"></a>
```

Comment:
Thanks for that fix!

Given that we "communicate" with pull requests, feel free to create separate PRs for such changes - as this is much easier/faster to review/accept than the other changes in this Pull Request.